compose-app/src/main/java/com/mapbox/maps/extension/compose/MapApplier.kt

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.maps.extension.compose
+
+
+import androidx.compose.runtime.AbstractApplier
+import com.mapbox.maps.MapControllable
+import com.mapbox.maps.plugin.delegates.MapPluginProviderDelegate
+
+internal interface MapNode {
+  fun onAttached() {}
+  fun onRemoved() {}
+  fun onCleared() {}
+}
```

Comment:
maybe add `onChanged`?

Reply:
actually I don't find use case for it yet, and we don't get onChanged notification from `AbstractApplier`